screens/ChatScreen.js

```diff
@@ -111,6 +113,17 @@ export default class ChatScreen extends React.Component {
     }
 }
 
+function mapStateToProps(state) {
```

Comment:
these are just conventions, you can ignore if you like.. i usually destructure everywhere so 'mapStateToprops({messages})' and then below you can do `return { messages }`

Reply:
Nice! I like that.

Reply:
Oooh wouldn't you have to refer to state.messages like:

mapStateToprops({state})
return { state.messages }

Oh I see this destructuring is crazy, it's actually returning state.messages in your above example?

Reply:
haha yeah, destructuring is crazy :)
basically you're passing a variable, and anything within that object can be destructured by putting it in {}. everything within the {} should be properties on the object you're passing in. so if you have your state like
```
state = {
   messages: [],
   user: {},
   chats: []
}
```
you could pass in `mapStateToProps({messages, user, chats})` and then in that function you can skip saying state.messages, state.user, state.chats.

Reply:
:100: Thanks